### PR TITLE
Updated readme with a few bits

### PR DESCRIPTION
**Pull request by forceedge**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +28,7 @@ It is recommended to place this code snippet in your website's template page. It
 [TODO] I think this is ok as we the `AT.send` if clients want to do anything based on the DOM, e.g.
 
 ```js
-jQuery(function () {
+jQuery(document).ready(function () {
```
Comment (jabclab) [inline README.md]: The previous would have worked ([docs](https://api.jquery.com/ready/)) but agreed it's best to be specific :smile_cat:

File: README.md [hunk 2]
```diff
@@ -64,20 +74,35 @@ For ad hoc requests you should send the data using the [JavaScript API](#api-usa
...
+        });
+      });
   });
 ```
 
 #### Sharepoints
-[TODO] DO THEN DOCUMENT
+```js
+jQuery(document).ready(function() {
+    document.querySelector('#social_button')
```
Comment (jabclab) [inline README.md]: Looks like a mix of 4 space and 2 space indentation

File: README.md [hunk 3]
```diff
@@ -94,48 +119,54 @@ Sends data to the Advocate Things API.
...
-document
-  .querySelector('img.banner')
-  .addEventListener('click', function handleBtnClick() {
-    AT.send({
-    	_at: { touchpoint: 'banner_click' }
-    }, function () {
-    	// Perform button's action
-    });
+jQuery(document).ready(function() {
+    document
```
Comment (jabclab) [inline README.md]: 4 and 2 space indentation

File: README.md [hunk 4]
```diff
@@ -295,10 +330,13 @@ Change any above occurrence of `gulp test` to `gulp test-min` in order to test
...
+In order to build the JS file use [gulp](http://gulpjs.com/) as follows:
 
 ```
 $ gulp build
 ```
 
 The built file will end up in `${clone}/dist/sdk.js`. It is not yet minified.
+
+## <a name="api-dependencies"></a>Dependencies
+- jQuery
```
Comment (jabclab) [inline README.md]: jQuery isn't strictly a dependency of the SDK (we really don't want it to be).

**Comment by jabclab** [comment]
Where we've wrapped examples in `jQuery(document).ready` it would be good to use the [vanilla JS](https://developer.mozilla.org/en-US/docs/Web/API/Document/readyState) instead (or just leave out in the code the fact that the DOM must be loaded before event listeners can be attached, a note could be added but we probably don't need it in every code snippet).

**Comment by forceedge** [comment]
In regards to the jQuery(document).ready If we want to be idiot proof i guess we need to get rid of the obvious mistakes, when someone who doesnt fully understand the code while copy pasting it into a html page and just assumes that this will just work as is, this will come up. But yeah if you want we can have a note saying this.

File: README.md (hunk 4 above)
Comment (forceedge) [inline README.md]: Is it used in the SDK anywhere at all? i.e if jQuery is not present would the SDK still work as intended?

File: README.md (hunk 3 above)
Comment (forceedge) [inline README.md]: Fair point, will make them 4

File: README.md (hunk 2 above)
Comment (forceedge) [inline README.md]: :( why did this happen...

File: README.md (hunk 4 above)
Comment (forceedge) [inline README.md]: Had a look through the SDK, you are right, jQuery is not a dependency. Il get rid of this.

**Comment by forceedge** [comment]
Had another thought, someone could be using something other than jQuery, this doesnt apply, will just put a not on these snippets that the element should be loaded prior to the call being made.

File: README.md [hunk 5]
```diff
@@ -3,60 +3,67 @@ Advocate Things JavaScript SDK [![Build Status](https://travis-ci.org/digitalani
...
+                user_id: '{{ user.id }}'
+           	},
+	       	// Metadata goes outside the _at object
+	       	page: {
+                id: '{{ page.id }}',
+                category: '{{ page.category }}'
+	      	}
+        };
+  	</script>
+    <script src="https://sdk.at.com/at-sdk-0.0.1.js?key={{ YOUR_KEY }}" type="text/javascript"></script>
```
Comment (jabclab) [inline README.md]: This `<script>` element is missing an `id="advocate-things-script"` attribute.

Comment (forceedge) [inline README.md]: Didn't know about that one sorry, what is it for?

**Comment by jabclab** [comment]
For the code examples I'd probably go for this style:

```js
document
  .querySelector('#my-selector')
  .addEventListener('hover', function myListener() {

  });
```

rather than:

```js
document
.querySelector('#my-selector')
.addEventListener('hover', function myListener() {

});
```

File: README.md (hunk 5 above)
Comment (jabclab) [inline README.md]: It's just an example which provides a minimal (but complete) working example :smile:.

Comment (forceedge) [inline README.md]: Cool will add that in :o:

**Pull request merged**